Makefile

```diff
@@ -23,17 +24,16 @@ default: test build
 
 .PHONY: prepare
 prepare:
-	go get -u github.com/kardianos/govendor
-	govendor sync
+	go get ./...
```

Comment:
Is this really necessary? Can we get rid of `prepare` step entirely with Go modules?

Reply:
Removed prepare phase completely